code-ready/crc
# Refactor logging code

Remove the file hook and set the logfile as the output destination
for logrus, this enables the daemon service in windows to put logs
in the log file, both from the machine side and the logs from logrus

In case of the daemon service in windows there is no stdout and stderr devices
and the logs from logrus doesn't appear in the logfile this commit sets the
logfile as the primary output for logrus, which seems to solve the issue.

### Testing
1. Remove the existing daemon (`crc cleanup`)
2. install daemon from this PR (`crc setup --enable-experimental-features`)
3. check ~/.crc/crcd.log for logs while interacting from the tray, it should logs that are from logrus
